frontend/reactnative/App.js

```diff
@@ -14,6 +14,19 @@ export default class App extends Component {
         location: null,
         errorMessage: null
     };
+	
+	constructor(props){
+		super(props);
+		
+		//Auth.wipeUserId();
+		Auth.loadUserId();
```

Comment:
call await on `Auth.loadUserId();` otherwise you're just adding startup time to the application

Reply:
Await is a reserved word, and can only be called in a asynchronous function. So it cannot be used here.